### ISSUE TITLE
Kraken, EXMO broken in 1.18.71

### ISSUE THREAD

**Issue by adegtyarev** [ccxt/ccxt#4392]
```
>>> import ccxt
>>> ccxt.__version__
'1.18.71'
>>> try:
...     t = ccxt.exmo().load_markets()
... except Exception as e:
...     print(getattr(e, 'message', repr(e))[:100])
...
ExchangeError('exmo GET https://exmo.me/en/docs/fees failed to decode json <!DOCTYPE html>\n<html cl

>>> try:
...     t = ccxt.kraken().load_markets()
... except Exception as e:
...     print(getattr(e, 'message', repr(e))[:100])
...
ExchangeError('kraken GET https://support.kraken.com/hc/en-us/articles//205893708-What-is-the-minimu
>>>
```

Looks like both failures were introduced with b906f6694c8e98c6b134ca1eba8900c87482e1a0 & dfb5b23ba2ef77b1a8b6099dfb311e8fe4375c73.

**Comment by kroitor** [comment]
Sorry for the inconvenience. It is now fixed in 1.18.72 (will be available for downloading in 5-10 minutes). Thx for reporting and for your attention!

**Comment by nevmerzhitsky** [comment]
Hi guys!

I'm not sure about the fix in 1.18.72, but starting from 1.18.75 `kraken.load_markets()` in `async_support` has broken up to 1.18.80. I haven't code example, but backtrace should be enough:

```
ccxt version: 1.18.75
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.7/threading.py", line 865, in run
    self._target(*self._args, **self._kwargs)
  File "/var/app/src/lib.py", line 44, in start_thread
    asyncio.run(cb(symbol_in, symbol_out))
  File "/usr/local/lib/python3.7/asyncio/runners.py", line 43, in run
    return loop.run_until_complete(main)
  File "/usr/local/lib/python3.7/asyncio/base_events.py", line 573, in run_until_complete
    return future.result()
  File "scrape.py", line 31, in init_and_start_scraping
    client = await init_client()
  File "scrape.py", line 62, in init_client
    await result.load_markets()
  File "/usr/local/lib/python3.7/site-packages/ccxt/async_support/base/exchange.py", line 174, in load_markets
    markets = await self.fetch_markets(params)
  File "/usr/local/lib/python3.7/site-packages/ccxt/async_support/kraken.py", line 266, in fetch_markets
    limits = await self.fetch_min_order_amounts()
  File "/usr/local/lib/python3.7/site-packages/ccxt/async_support/kraken.py", line 245, in fetch_min_order_amounts
    parts = html.split('<td class="wysiwyg-text-align-right">')
AttributeError: 'NoneType' object has no attribute 'split'
```

**Comment by kroitor** [comment]
@nevmerzhitsky yep, that's my bad, a leftover in the async branch, fixed it in 1.18.81. Thx for the feedback!
